### PR TITLE
Feature/log version number on startup

### PR DESCRIPTION
**Pull request by JamesGardiner**
This is a pull request to add functionality that logs the current version number on service startup. The `__version__` variable in `console/init.py` contains the release value, which will need to be manually updated on a new release being cut. The version is logged at INFO level.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ### Unreleased
+  - Log version number on startup
   - Fix handling of None responses in remote call
   - Add SFTP support
   - Change `nack for retry` to `nack` for SDX logging
   - Change `status_code` to `status` for SDX logging
 
 ### 1.0.0 2017-02-2016
+=======
```
Comment (necrophonic) [inline CHANGELOG.md]: Looks like a merge conflict artefact has snuck in

File: app/consumer.py [hunk 2]
```diff
@@ -60,6 +61,7 @@ def on_message(self, unused_channel, basic_deliver, properties, body):
 
 def main():
     logger.debug("Starting consumer")
+    logger.info("Current version: {}".format(__version__))
```
Comment (necrophonic) [inline app/consumer.py]: Why not a single log line (info level) with the event and the version, e.g.
```
logger.info("Starting consumer", version=__version__)
```
(If you can embed the version like that - you may need the format stuff)

**Review by necrophonic — DISMISSED** [review]

File: app/consumer.py (hunk 2 above)
Comment (JamesGardiner) [reply app/consumer.py]: Good point - should have checked Splunk's ingesting preferences first! I'll update this and other sdx-* services accordingly.

**Comment by JamesGardiner** [comment]
@necrophonic - all requested changes made. Ok to merge?

**Review by eldeal — APPROVED** [review]

**Pull request merged**